File: CS50's_Introduction_to_CS/Week1/Problem_set_1/cash.c
//
// Created by saksham on 16-06-2021.
//

#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    //  (25¢), dimes (10¢), nickels (5¢), and pennies (1¢).
    float dollars;
    do
    {
        dollars = get_float("Change owed: ");
    }
    while (dollars < 0);
    int input = round(dollars * 100);

    int coins = 0;

    while (input > 0)
    {
        if (input >= 25)
        {
            input = input - 25;
        }
        else if (input >= 10)
        {
            input = input - 10;
        }
        else if (input >= 5)
        {
            input = input - 5;
        }
        else
        {
            input = input - 1;
        }
        coins++;
    }
    printf("%i\n", coins);
}

File: CS50's_Introduction_to_CS/Week6/Problem_set_6/cash/cash.py
from cs50 import get_float

while True:
    # Getting value from user:
    value = get_float("Change owed: ")
    if value > 0:
        break

value = round(value * 100)
coins = 0
while value > 0:
    if value >= 25:
        value -= 25
    elif value >= 10:
        value -= 10
    elif value >= 5:
        value -= 5
    else:
        value -= 1
    coins += 1
print(coins)